pkg/util/hostname_docker.go

```diff
@@ -14,41 +14,33 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/hostname"
-	"github.com/DataDog/datadog-agent/pkg/util/hostname/validate"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 func getContainerHostname(ctx context.Context) string {
+	var err error
 	if config.IsFeaturePresent(config.Kubernetes) {
 		// Cluster-agent logic: Kube apiserver
-		if getKubeHostname, found := hostname.ProviderCatalog["kube_apiserver"]; found {
-			log.Debug("GetHostname trying Kubernetes trough API server...")
```

Comment:
could we keep this log line?

Reply:
oh I can see `GetHostname` is logging this instead, in this case we should get rid of `log.Debug("GetHostname trying Docker API...")` below to avoid duplicating the same log